app.kaiteki.Kaiteki.yml

```diff
@@ -0,0 +1,43 @@
+app-id: app.kaiteki.Kaiteki
+runtime: org.gnome.Platform
```

Comment:
does this really need the GNOME runtime?

Reply:
Kaiteki requires libhandy which is not included in the freedesktop runtime unless there is another runtime which includes it.

Reply:
Is that the only thing?

libhandy-0 or libhandy-1? It should not difficult to build on freedesktop-sdk. It requires glib2 and gtk3 both are present in the fsdk. You can disable vala (if you don't need it) and docs/glade.